composer.json

```diff
@@ -36,11 +36,11 @@
         "ramsey/collection": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7",
+        "phpunit/phpunit": "^8",
         "phpstan/phpstan": "^0.11.2",
         "friendsofphp/php-cs-fixer": "^2.14",
         "symfony/var-dumper": "*",
-        "symfony/cache": "*"
+        "symfony/cache": "^3.4"
```

Comment:
It seems that the `symfony/cache` component doesn't have `Symfony\Component\Cache\Simple\ArrayCache` class since version is `4.0+`.

It should use `^3.4` version to ensure that the `Symfony\Component\Cache\Simple\ArrayCache` class worked fine on `PHPUnit` tests execution.